index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+  <head>
+    <title>this is title</title>
+    <link rel="stylesheet" href="mystyle.css">
+  </head>
+  <body>
+    <div>
+    <p>
+      <span class="ex">
+        This is a <br>
+      </span>
```

Comment:
I'm not sure if nesting `<br>` inside a `<span>` is typical.  Maybe opt for:
```
<span>
  This is a
</span>
<br />
```

The `<br>` tag can be self-closing.  Not quite sure what the benefit of self-closing are, but it feels more proper.
http://xahlee.info/js/html5_non-closing_tag.html